x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { lazy } from 'react';
+import { suspendedComponentWithProps } from '../../lib/suspended_component_with_props';
+
+export const AlertsTable = suspendedComponentWithProps(lazy(() => import('./alerts_table')));
```

Comment:
did you consider exporting both a non-lazy and lazy version of the alert table?

Reply:
I'm not sure if that'd cause bundle size issues, which is why it's lazy in the first place. I can investigate if that's a requirement.